### ISSUE TITLE
Error: E3006 Resource types 'AWS::WorkSpaces::WorkspaceBundle' and 'AWS::WorkSpaces::Directory' do not exist in 'eu-west-2'

### ISSUE THREAD

**Issue by martbrooks** [aws-cloudformation/cfn-lint#3864]
### CloudFormation Lint Version

cfn-lint 1.20.2

### What operating system are you using?

Debian + homebrew

### Describe the bug

An AWS CloudFormation template that uses the above resource types fails to lint as follows:

$ cfn-lint AWSWorkspaces.yaml 
E3006 Resource type 'AWS::WorkSpaces::WorkspaceBundle' does not exist in 'eu-west-2'
AWSWorkspaces.yaml:7:5

E3006 Resource type 'AWS::WorkSpaces::Directory' does not exist in 'eu-west-2'
AWSWorkspaces.yaml:29:5

These resource types are available in eu-west-2, so far as I know.  Googling suggested that I should run `cfn-lint --update-specs` which I have done with no change to the error.

### Expected behavior

cfn-lint should not produce this error, at least for eu-west-2.

### Reproduction template

```yaml
AWSTemplateFormatVersion: 2010-09-09
Description: AWS Cloudformation template to create and configure an AWS Workspaces environment.


Resources:
  WorkspaceBundle:
    Type: AWS::WorkSpaces::WorkspaceBundle
    Properties:
      BundleDescription: "Standard Bundle"
      ComputeType: "STANDARD"
      Name: "StandardBundle"
      ImageId: "wsb-xxxxxxxxx"
      RootStorage:
        Capacity: 80
      UserStorage:
        Capacity: 10
        SupportsOverwrite: true
      WorkspaceImage:
        OperatingSystem: "WINDOWS_10"
        Type: "WINDOWS"
        Description: "Windows 10 Desktop"
        Name: "Windows10Desktop"
        RequiredTenancy: "DEDICATED"
      Tags:
        - Key: "Name"
          Value: "StandardBundle"

  WorkspaceDirectory:
    Type: AWS::WorkSpaces::Directory
    Properties:
      DirectoryName: "MyDirectory"
      DirectoryType: "SimpleAD"
      SubnetIds:
        - "subnet-xxxxxxxxx"
      WorkspaceSecurityGroupId: "sg-xxxxxxxxx"
      Tags:
        - Key: "Name"
          Value: "MyDirectory"

  Workspace:
    Type: AWS::WorkSpaces::Workspace
    Properties:
      BundleId: !Ref WorkspaceBundle
      DirectoryId: !Ref WorkspaceDirectory
      UserName: "user1"
      UserVolumeEncryptionEnabled: true
      VolumeEncryptionKey: "arn:aws:kms:us-east-1:123456789012:key/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
      Tags:
        - Key: "Name"
          Value: "user1"
```

**Comment by kddejong** [comment]
I don't see these resource types in our [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/AWS_WorkSpaces.html).  I may be missing something here.

I don't really see a workspace API for creating a directory service. Looks like there is an ability to register an existing directory to use a workspace but that looks covered under the `AWS::WorkSpaces::Workspace` resource.

I see this [issue](https://github.com/aws-cloudformation/cloudformation-coverage-roadmap/issues/1914) for the WorkspaceBundle resource.

**Comment by martbrooks** [comment]
I was cribbing from examples from Google.   I would expect a different error for "this does not exist here" versus "this does not exist at all."
